src/sentry/static/sentry/app/components/events/interfaces/contextLine.jsx

```diff
@@ -18,11 +19,24 @@ const ContextLine = function(props) {
   }
 
   return (
-    <li className={liClassName} key={line[0]}>
-      <span className="ws">{lineWs}</span>
-      <span className="contextline">{lineCode}</span>
-      {isActive && <OpenInButton filename={filename} lineNo={line[0]} group={group} />}
-    </li>
+    <React.Fragment>
+      {isActive ? (
+        <OpenInButton
```

Comment:
This is kind of confusing with the multiple ways we can render a context line item, and `OpenInButton` having rendering logic for a line item.

Reply:
I would imagine it to look like ContextLine renders `<OpenInButton><ContextLineItem /></OpenInButton>` - where `OpenInButton` is only responsible for the "open in button" and logic around that and not directly responsible for rendering the line item.